src/components/dropdown/interfaces.ts

```diff
@@ -7,6 +7,11 @@ export interface OptionRendererProps<T> {
   data: T;
 }
 
+export type HTMLInputProps = Omit<
+  React.HTMLProps<HTMLInputElement>,
+  'onChange'
+>;
```

Comment:
Looks like we lost the 'value' from `Omit<React.HTMLProps<HTMLInputElement>, 'onChange' | 'value'>` or did I miss something 😂

Reply:
the value is already part of our props @AliciaSymphony (check multimodeprops and singlemodeprops)